miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/rest/BoxRestController.java

```diff
@@ -630,6 +630,25 @@ private static boolean isRealBarcode(BoxScan scan, String barcode) {
     return getBoxDtoWithBoxables(boxId);
   }
 
+  @PostMapping("/box/{boxId}/bulk-remove")
+  public @ResponseBody BoxDto removeMultipleItems(@PathVariable long boxId, @RequestBody List<String> positions) throws IOException {
+    Box box = getBox(boxId);
+    for (String position : positions) {
+      box.getBoxPositions().remove(position);
```

Comment:
Could you add a check that there is an item in the position before removing it? (I selected a column that had a few blank positions and clicked "discard selected" and it returned `Internal Server Error: No item in the specified box position` and when I closed the dialog the page didn't refresh so it looked like nothing had happened, but when I manually refreshed the page it turned out that the selected items had actually been discarded)

Reply:
Oops, yes. I was originally returning 300 bad request if a position was empty, but that was pointless/annoying. Forgot to still check. Fixed now